website/docs/patterns/form-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,269 @@
+!!!
+
+Common sets of fields can include:
+
+- Credit card information: card number, expiration data, security code
+- Multi-line address field
+- First and last name
+
+## Alignment
+
+Fields organized in a horizontal set should be aligned to the baseline of each element to account for fields with helper text.
+
+![Baseline alignment within a set](/assets/patterns/form-patterns/baseline-alignment.png =500x*)
+
+## Layout
+
+### Single-column
+
+If a form uses a single-column layout, consider setting a maximum width on the form; this can generally be best achieved by using a size or unit that is **relative** the page, viewport, or container size. This can commonly be achieved by using a viewport width unit (`vw`), percentage width (e.g., `50%`), or a character unit (`ch`).
+
+We recommend _most_ forms use a single column layout as it makes the information within the form easier to parse for the user and reinforces the sequential nature of filling out a form.
+
+!!! Info
+
+The web is a fluid medium. A relative width will adjust based on the viewport and container width. Designing and building a form with a maximum and minimum width will ensure a graceful expansion and contraction within the viewport.
+!!!
+
+### Multi-column
+
+Horizontal sets of form elements creates a multi-column layout within the form.
+
+!!! Do
+
+Use a consistent number of columns through the form.
+
+![Consistent number of columns](/assets/patterns/form-patterns/multi-column-consistent-columns.png =500x*)
+!!!
+
+!!! Dont
+
+Don't organize a set of fields in two columns and another set in three columns.
+
+![Inconsistent number of columns](/assets/patterns/form-patterns/multi-column-inconsistent-columns.png =500x*)
+!!!
+
+!!! Dont
+
+Don't exceed more than three fields in a horizontal set. In most scenarios it's best to limit the number of columns in a form or section to two.
+
+![Three field maximum in a set](/assets/patterns/form-patterns/multi-column-three-fields-max.png =500x*)
+!!!
+
+!!! Do
+
+Stack fields vertically when the width of the viewport and form container shrinks.
+
+![Stack fields](/assets/patterns/form-patterns/multi-column-stacking.png =350x*)
+!!!
+
+The width of the overall form can impact multi-column layouts. Use set organization logically based on the layout, width of the page, and overall UX strategy in the application.
+
+## Text
+
+In sections consisting of multiple blocks of text, use an 8px vertical gap between elements.
+
+![Example of a text section](/assets/patterns/form-patterns/text-section.png =600x*)
+
+Text elements within a form should use logical, step-based sizing to reinforce hierarchy within sections and the form itself. While specifics around type hierarchy should be determined at the application level, adhering to these guidelines will help establish consistency at the page level and when constrained within another element or component.
+
+## Actions
+
+### Button sets
```

Comment:
Do you anticipate more content being added to the "Actions" section? If not, it may help reduce the headings displayed in the sidecar to decide between "Actions" or "Button sets" as the singular heading for this content.